KeyShot/5.9.0/config/media_manager_5/fields/keyshot_metafields.dcl

```diff
@@ -0,0 +1,50 @@
+data string_metafield title {
+    item_guid = '5eb3eefc-a043-410f-89b0-29ed3ef37078'
+}
+
+resource configservice_multi_string_config_field creative_cloud_connector_adobe_links {
```

Comment:
The name of this resource seems off 🤔 
Maybe `keyshot_metafields` would be more appropriate?